apps/demo-app/src/views/HomeView.vue

```diff
@@ -278,10 +278,37 @@ const currentPage = ref(1);
           label="Show toast"
           @click="toast.show({ headline: 'Example toast', color: 'success' })"
         />
-
-        <OnyxTooltip v-if="show('OnyxTooltip')" text="Example tooltip text">
-          Hover me to show tooltip
-        </OnyxTooltip>
+        <OnyxHeadline is="h2">Tooltip (auto alignment)</OnyxHeadline>
+
+        <div
+          :style="{
+            display: 'flex',
+            justifyContent: 'space-between',
+            width: '101%',
+            marginTop: '2rem',
+          }"
+        >
+          <OnyxTooltip v-if="show('OnyxTooltip')" text="Example tooltip text">
+            <template #default="{ trigger }">
+              <OnyxButton label="Left" v-bind="trigger" />
+            </template>
+          </OnyxTooltip>
+          <OnyxTooltip v-if="show('OnyxTooltip')" text="Example tooltip text">
+            <template #default="{ trigger }">
+              <OnyxButton label="Center" v-bind="trigger" />
+            </template>
+          </OnyxTooltip>
+          <OnyxTooltip v-if="show('OnyxTooltip')" text="Example tooltip text">
+            <template #default="{ trigger }">
+              <OnyxButton label="Center" v-bind="trigger" />
+            </template>
+          </OnyxTooltip>
+          <OnyxTooltip v-if="show('OnyxTooltip')" text="Example tooltip text">
+            <template #default="{ trigger }">
+              <OnyxButton label="Right" v-bind="trigger" />
+            </template>
+          </OnyxTooltip>
+        </div>
```

Comment:
- please move styles to a class
- if you bundle the examples, you can set v-if just once on the div :)
```suggestion
        <div class="tooltip-demo" v-if="show('OnyxTooltip')" >
          <OnyxTooltip text="Example tooltip text">
            <template #default="{ trigger }">
              <OnyxButton label="Left" v-bind="trigger" />
            </template>
          </OnyxTooltip>
          <OnyxTooltip text="Example tooltip text">
            <template #default="{ trigger }">
              <OnyxButton label="Center" v-bind="trigger" />
            </template>
          </OnyxTooltip>
          <OnyxTooltip text="Example tooltip text">
            <template #default="{ trigger }">
              <OnyxButton label="Center" v-bind="trigger" />
            </template>
          </OnyxTooltip>
          <OnyxTooltip text="Example tooltip text">
            <template #default="{ trigger }">
              <OnyxButton label="Right" v-bind="trigger" />
            </template>
          </OnyxTooltip>
        </div>
```